crates/astria-sequencer/src/app/tests_execute_transaction.rs

```diff
@@ -1100,3 +1109,234 @@ async fn transaction_execution_records_fee_event() {
             .into()
     );
 }
+
+#[tokio::test]
+async fn ensure_correct_block_fees_transfer() {
+    let mut app = initialize_app(None, vec![]).await;
+    let mut state_tx = StateDelta::new(app.state.clone());
+    state_tx.put_transfer_base_fee(1).unwrap();
+    app.apply(state_tx);
+
+    let alice = get_alice_signing_key();
+    let bob_address = astria_address_from_hex_string(BOB_ADDRESS);
+    let actions = vec![
+        TransferAction {
+            to: bob_address,
+            amount: 1000,
+            asset: nria().into(),
+            fee_asset: nria().into(),
+        }
+        .into(),
+    ];
+
+    let tx = UnsignedTransaction {
+        params: TransactionParams::builder()
+            .nonce(0)
+            .chain_id("test")
+            .build(),
+        actions,
+    };
+    let signed_tx = Arc::new(tx.into_signed(&alice));
+    app.execute_transaction(signed_tx).await.unwrap();
+
+    let total_block_fees: u128 = app
+        .state
+        .get_block_fees()
+        .await
+        .expect("failed to get block fees")
```

Comment:
```suggestion
        .expect("can get block fees")
```

`expect` is used with the success case statement, ie. what you expect to happen. also, in this case, you can probably just use `unwrap` anyways, as it's clear what the issue is if `get_block_fees` fails